routing/geometry.hpp

```diff
@@ -89,22 +89,30 @@ class GeometryLoader
       std::string const & fileName, std::shared_ptr<VehicleModelInterface> vehicleModel);
 };
 
+/// \brief This class supports loading geometry of roads for routing.
+/// \note Loaded information about road geometry is kept in evicted cache |m_featureIdToRoad|.
+/// On the other hand methods GetRoad() and GetPoint() return geometry information by reference.
+/// The reference is valid until the next call of GetRoad() or GetPoint() because the cache
```

Comment:
s/The reference is valid until the next call/The reference may be invalid after the next call/
or
The reference is generally not valid after the next call